content/herman/index.rst

```diff
@@ -12,6 +13,14 @@ project:
 contributors:
   - author: 'oddbird'
     role: 'creators'
+brag: |
+  Created by OddBird,
+  Herman provides
+  **fully-integrated style documentation** –
+  from Sass abstracts to
+  nunjucks and vue components –
```

Comment:
Hm, do we want to advertise this before we've built it?

Reply:
We could pull it out for now?

Reply:
Yeah, personally I'd pull out `Vue components` for now. Could also call out colors/fonts specifically? In my mind they're a big part of Herman, and aren't obvious from "Sass abstracts".